src/components/NewHathorButton.js

```diff
@@ -64,19 +65,19 @@ const style = StyleSheet.create({
   wrapper: {
     height: 48,
     borderRadius: 8,
-    backgroundColor: '#000',
+    backgroundColor: COLORS.textColor,
     alignSelf: 'stretch',
   },
   wrapperDisabled: {
-    backgroundColor: 'rgba(0, 0, 0, 0.05)',
+    backgroundColor: COLORS.textColorShadowLighter,
```

Comment:
Changed from opacity `0.05` to `0.1`

Reply:
Why?

Reply:
Reverted to `0.05` opacity on [a6b16fd](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/327/commits/a6b16fdf489a4652038ac614da23fe5485e62020)